### DISCUSSION TITLE
How to disable MQTT tele/%topic/STATE telegram?

### DISCUSSION THREAD

**Discussion by Mugridge** [arendst/Tasmota#22088] [Q&A]
The command  mosquitto_sub -h localhost -t "#" -v on my Raspberry Pi shows me that every Gosund SP111 device with Tasmota is sending two MQTT telegrams, one with the tele/%topic/SENSOR header with the values I need and one with the  tele/%topic/STATE header which I don't need. How can I switch off the sending of the STATE telegram? Thanks.

**Comment by sfromis** [comment]
The way is to subscribe to what you want, and ignore what you don't have a current use for. A **minuscule** amount of network traffic is nothing to worry about.

**Comment by Mugridge** [comment]
Ok, it's not possible. So I don't have to try anymore. Thx.
